docs/PFC-test-plan.md

```diff
@@ -80,7 +131,8 @@ Keysight ports are connected with SONiC switch as shown in the illustration abov
 
 ### DUT configuration
 
-- PFC watch dog is disabled
+- [PFC watchdog](https://github.com/Azure/SONiC/wiki/PFC-Watchdog-Design) is disabled. The PFC functionality under test here is when PFC watchdog is not active. Device Under Test should drop traffic with lossless priority upon receiving PFC PAUSE storm for the same priority even in absence of PFC watchdog. Hence its desired to keep watchdog disabled.
```

Comment:
Describe the reason to disable PFC watchdog as follows:

If a queue has been paused for long time (e.g., several hundreds of milliseconds), PFC watchdog will be triggered to disable PFC and drop packets. Since we will generate continuous PFC pause frames (which is unlikely to happen in production) to test PFC functionality, we decide to disable PFC watchdog.

Reply:
Thanks added this one